spec/utils/generate-redis-connection-expr.spec.ts

```
@@ -0,0 +1,161 @@
+import { describe, it, expect, beforeEach, afterEach } from 'vitest'
+import { generateRedisConnectionExpr } from '../../src/utils/generate-redis-connection-expr'
+
+// Helper: evaluate the generated expression in current process context
+function evalExpr(expr: string): unknown {
+  return eval(expr)
```

Biome (2.3.13)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)